src/lib/moment/get-set.js

```diff
@@ -19,29 +17,72 @@ export function makeGetSet(unit, keepTime) {
 }
 
 export function get(mom, unit) {
-    return mom.isValid()
-        ? mom._d['get' + (mom._isUTC ? 'UTC' : '') + unit]()
-        : NaN;
+    if (!mom.isValid()) {
+        return NaN;
+    }
+
+    var d = mom._d,
+        isUTC = mom._isUTC;
+
+    switch (unit) {
+        case 'Milliseconds':
+            return isUTC ? d.getUTCMilliseconds() : d.getMilliseconds();
+        case 'Seconds':
+            return isUTC ? d.getUTCSeconds() : d.getSeconds();
+        case 'Minutes':
+            return isUTC ? d.getUTCMinutes() : d.getMinutes();
+        case 'Hours':
+            return isUTC ? d.getUTCHours() : d.getHours();
+        case 'Date':
+            return isUTC ? d.getUTCDate() : d.getDate();
+        // case 'Day': return isUTC ? d.getUTCDay() : d.getDay(); // Not used
```

Comment:
Why is this unused?

Reply:
Sorry for leaving the comments in. There no problem or anything. It's just that that `getSetDayOfWeek` has it's own implementation instead of `makeGetSet('Day', true)` and it doesn't call `get(this, 'Days')` like `getSetMonth` does.

This is actually the reason `get('day')` performance was higher than the rest :)
Anyway, now that wouldn't matter with new code, so I'll make sure `get(this, '...')` is used where is can be used as part of this commit. That looks like 1 LOC in a couple of files.

Reply:
Done! 
Now using `get` in `getSetDayOfWeek`.
I ended up leaving the comments in `set()` and added the reason we cannot do this for `setMonth`.
It can be used there too, if I implement separate private functions instead of `get` and `set`.
I'd love to do that too, but that's a bigger move and I think should be discussed separately.